src/test/java/com/iexec/blockchain/chain/QueueServiceTests.java

```diff
@@ -167,8 +183,10 @@ private void waitQueueReady(AtomicBoolean queueReady) {
     //region BlockchainAction
     @Test
     void compareBlockchainActionAgainstNull() {
-        QueueService.BlockchainAction lowPriorityAction = new QueueService.BlockchainAction(() -> {}, false);
-        QueueService.BlockchainAction highPriorityAction = new QueueService.BlockchainAction(() -> {}, true);
+        QueueService.BlockchainAction lowPriorityAction = new QueueService.BlockchainAction(() -> {
+        }, false);
+        QueueService.BlockchainAction highPriorityAction = new QueueService.BlockchainAction(() -> {
+        }, true);
```

Comment:
Did you add the line breaks or were they added by auto-reformating ? Don't you find it easier to read with no line break (the lines won't be longer, without the line break, than the `238` and `239` ones) ?

Lines `238` and `239`:
```java
QueueService.TaskWithPriority<Runnable> lowPriorityTask = new QueueService.TaskWithPriority<>(lowPriorityAction);
QueueService.TaskWithPriority<Runnable> highPriorityTask = new QueueService.TaskWithPriority<>(highPriorityAction);
```
```suggestion
        QueueService.BlockchainAction lowPriorityAction = new QueueService.BlockchainAction(() -> {}, false);
        QueueService.BlockchainAction highPriorityAction = new QueueService.BlockchainAction(() -> {}, true);
```

Reply:
It is the auto-formatting feature which enforces this rule